main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"github.com/luciodesimone/golang-bootcamp/db"
+	"log"
+)
+
+func main() {
+	stout := db.Beer{
+		Desc:        "Creamy black dense beer with coffee notes, ideal before dinner",
+		IBU:         42,
+		AlcoholCont: 8,
+		SRM:         40,
+		AvgScore:    92,
+	}
+
+	bitter := db.Beer{
+		Desc:        "dark amber light beer, ideal for summer",
+		IBU:         57,
+		AlcoholCont: 5.5,
+		SRM:         10,
+		AvgScore:    63,
+	}
+
+	barley := db.Beer{
+		Desc:        "Strong, dense red/dark beer, ideal for late night with friends",
+		IBU:         70,
+		AlcoholCont: 12,
+		SRM:         22,
+		AvgScore:    86,
+	}
+
+	c := db.Connection()
+
+	s := c.Create(stout)
```

Comment:
If you could create a function that expects the db interface and does the same that you are doing from here and then call it again with a second implementation of the db interface that would showcase one of the best attributes of interfaces.

What I mean is, something like 

`
c_mongo:= db.Connection()
c_msql:=dbmsql.Connection()

operate(c_mongo, beers)
operate(c_msql, beers)
`